netlib/http/cookies.py

```diff
@@ -263,27 +269,69 @@ def refresh_set_cookie_header(c, delta):
     return ret
 
 
-def is_expired(cookie_attrs):
+def get_expiration_ts(cookie_attrs):
     """
-        Determines whether a cookie has expired.
+        Determines the time when the cookie will be expired.
 
-        Returns: boolean
-    """
+        Considering both 'expires' and 'max-age' parameters.
 
-    # See if 'expires' time is in the past
-    expires = False
+        Returns: timestamp of when the cookie will expire.
+                 None, if no expiration time is set.
+    """
     if 'expires' in cookie_attrs:
         e = email.utils.parsedate_tz(cookie_attrs["expires"])
         if e:
-            exp_ts = email.utils.mktime_tz(e)
+            return email.utils.mktime_tz(e)
+
+    elif 'max-age' in cookie_attrs:
+        try:
+            max_age = int(cookie_attrs['Max-Age'])
```

Comment:
There's no policy, i guess because the [`_kconv`](https://github.com/dufferzafar/mitmproxy/blob/har/netlib/http/cookies.py#L179) function converts these keys to lowercase.

IMO, we should keep using mixed cases just to show that they don't matter.